src/lib/style/components/evaluation_queue/_calendar_with_icon_input_group.scss

```diff
@@ -21,4 +21,13 @@
       }
     }
   }
+  .css-12wm9o4-MuiInputBase-root-MuiOutlinedInput-root {
+    margin-top: 0;
+    padding: 8px;
+    border: 1px #dcdcdc solid;
+    border-radius: 2px;
+    .css-ihdtdm {
+      width: 0;
+    }
+  }
```

Comment:
These CSS selectors are auto-generated and will change, so this won't be applied.